http/src/main/java/org/apache/pekko/http/javadsl/settings/OversizedSseStrategy.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.http.javadsl.settings;
+
+/**
+ * Strategy for handling oversized SSE messages that exceed the configured max-line-size.
+ *
+ * @since 1.3.0
+ */
+public enum OversizedSseStrategy {
```

Comment:
How about adding the getInstance method in the Scala OversizedSseStrategy, this will keep the same as pekko, otherwise, there will be two OversizedSseStrategys shows in IDE

Reply:
See Pekko-core `org.apache.pekko.stream.OverflowStrategy`

Reply:
right, so that's not in a `javadsl`/`scaladsl` package but moves it to the common parent package. https://github.com/apache/pekko/blob/main/CONTRIBUTING.md#java-apis-in-pekko does not provide a clear preference, and I don't think I have a clear preference either.